packages/widgets/src/lib/components/InferenceWidget/InferenceWidget.svelte

```diff
@@ -79,9 +79,11 @@
 	};
 
 	$: widgetComponent =
-		model.pipeline_tag && model.pipeline_tag in WIDGET_COMPONENTS
-			? WIDGET_COMPONENTS[model.pipeline_tag as keyof typeof WIDGET_COMPONENTS]
-			: undefined;
+		model.pipeline_tag === "text-generation" && model.tags?.includes("conversational")
```

Comment:
maybe also `model.pipeline_tag === text2text-generation` btw @osanseviero no?

Reply:
Not needed imo. https://huggingface.co/models?pipeline_tag=text2text-generation&other=conversational&sort=trending only has 29 models and they don't have templates (vs 9300 public models in https://huggingface.co/models?pipeline_tag=text-generation&other=conversational&sort=trending )

Reply:
(also moot if we merge text2text-generation into text-generation like i'd like to 😁)